items/item.go

```diff
@@ -92,6 +104,11 @@ func (s *Store) Item(id string) (*Item, error) {
 	if result != nil && len(result.Versions) > 0 {
 		return result, nil // a complete item record
 	}
+	// if item not in cache, and useStore disabled, return ErrNoStore Error error
+	if result == nil && s.useStore == false {
```

Comment:
I didn't know id I'd ever get the case result != nil ,
len(result.Versions) = 0

I will remove it.

On Thu, Aug 25, 2016 at 12:19 PM, Don Brower <notifications@github.com>
wrote:

> In items/item.go
> <https://github.com/ndlib/bendo/pull/136#discussion_r76275941>:
>
> > @@ -92,6 +104,11 @@ func (s *Store) Item(id string) (*Item, error) {
> >  	if result != nil && len(result.Versions) > 0 {
> >  		return result, nil // a complete item record
> >  	}
> > +	// if item not in cache, and useStore disabled, return ErrNoStore Error error
> > +	if result == nil && s.useStore == false {
>
> result will always be nil here. you can remove that condition.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76275941>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTDtNCNDs8qkedI01mdPVcbOXwTq8ks5qjcCVgaJpZM4JsckT>
> .
>